packages/webservice/app/src/main/resources/demo-configs/models/tables/DemoTables.hjson

```diff
@@ -53,33 +61,39 @@
         }
         {
           name: film_rating
+          category: Attributes
           type: TEXT
           definition: '{{rating}}'
         }
         {
           name: genres
+          category: Attributes
           type: TEXT
           definition: '{{listed_in}}'
         }
         {
           name: description
+          category: Attributes
           type: TEXT
           definition: '{{description}}'
         }
       ]
       measures: [
         {
           name: count
+          category: Stats
```

Comment:
Should we have three categories with one metric each or one category with three metrics?

Reply:
Based on this data and since it is only a demo, it is more clear (For this) to have 3 categories with one metic each, since season only applies to show and duration to movies ... but if you do not feel strong of it, we can have:
Stats -|
           |  (Change metic name to point to Movie or TV)